packages/wc-compass/src/layouts/about/_about.scss

```diff
@@ -0,0 +1,81 @@
+div.cdg-about-section {
+  display: flex;
+  flex-direction: row;
+  color: var(--grey100);
+  padding-top: 32px;
+  padding-bottom: 32px;
+  container-name: CdgAbout;
+  container-type: inline-size;
+  justify-content: space-between;
+
+  .cdg-about-left {
+    width: 50%;
+  }
+
+  .cdg-about-right {
+    background: var(--accent);
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+  }
+}
+
+@media screen and (max-width: 960px) {
+  div.cdg-about-section {
+    flex-direction: column;
+  }
+
+  .cdg-about-left {
+    width: 100% !important;
+    overflow: hidden;
+  }
+}
+
+.cdg-about-right {
+  display: flex;
+  flex-direction: column;
+  color: white;
+
+  .cdg-about-header {
+    font-size: 3rem;
+    line-height: 3.5rem;
+    margin: 0;
+  }
+
+  &-wrap {
+    max-width: 544px;
+    padding: 142px 100px;
+  }
+
+  &-content {
+    padding: 1rem 0;
+  }
+
+  .cdg-about-statistical {
+    margin-top: 3rem;
+    p {
```

Comment:
Don't use specific tag name like this, in the future you may have more than 1 p tag in here, so this css will effects on both.

Reply:
fixed